hash/megadriv.xml

```diff
@@ -24047,7 +24093,7 @@ Black screen
 	</software>
 
 	<software name="pyramidgnk" cloneof="pyramid">
-		<description>Pyramid Magic (Japan, Game no Kandume MegaCD rip)</description>
+		<description>Pyramid Magic (Japan, ripped from Game no Kanzume Vol. 1)</description>
```

Comment:
I'm not sure this change is for the best, I feel like it strips out a context plus I don't like the form, same below.

Reply:
Regarding the form, if you mean the "ripped from", I added that since games ripped from bootleg cartridges are also marked as such. Should I just remove all instances of "ripped from"?

Reply:
If you do strip details from the description, you should put those details into a comment in the entry so that context is kept in some way. That information can sometimes end up being useful in a future situation (e.g. "When was this image replaced and why?")

Reply:
@Firehawke:
> It actually adds missing context (specifies which volume) and removes unnecessary info (both Game no Kanzume volumes 1 and 2 were released exclusively on the Mega-CD so there's no need to specify).

Reply:
So should I remove all instances of "ripped from" across the softlist?